### DISCUSSION TITLE
Add {if editable} to Channel:Entries tag

### DISCUSSION THREAD

**Discussion by nep** [ExpressionEngine/ExpressionEngine#4376] [Feature Requests]
### Suggest an idea for ExpressionEngine

**Is your feature request related to a problem? Please describe.**

Currently, there's an {if editable} option for Comments, but not for entries.
https://docs.expressionengine.com/latest/comment/form.html#editing-permissions

It would be a great conditional to add to Channel Entries as well.
https://docs.expressionengine.com/latest/channels/entries.html#cp_edit_entry_url

This would eliminate having to look up someone's role(s) and then permissions. This would make custom front-end editing much easier to build.

**Comment by intoeetive** [comment]
Implemented slightly differently : https://github.com/ExpressionEngine/ExpressionEngine/pull/4455

Instead of adding extra variable, [cp_edit_entry_url](https://docs.expressionengine.com/latest/channels/entries.html#cp_edit_entry_url) is empty if the member has no access, so you can also check with `{if cp_edit_entry_url}`

**Comment by nep** [comment]
So now cp_edit_entry_url is either 1) false if no permission, or 2) a CP URL for anyone who CAN edit, instead of being either 1) false if logged out or 2) a CP URL for anyone logged in, but that may not be accessible by that specific member?

Does that mean this code would be what is needed to identify someone logged in who can edit a specific entry BUT NOT access the control panel? `{if cp_edit_entry_url AND ! cp_url}`

Regardless, I still think there's a benefit in having two variables instead of overloading one, and also in having consistency in variables between comments and entries.

**Reply by intoeetive** [reply]
The value of cp_edit_entry_url did no change for the users with access to edit that specific entry. If the user has no access, it's empty string
